contracts/strategies/pangolin/strategy-png-spore-png.sol

```diff
@@ -0,0 +1,94 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.6.7;
+
+contract StrategyPngSporePngLp is StrategyPngFarmBase {
+    // Token addresses
+    address public png_spore_png_rewards = 0x12A33F6B0dd0D35279D402aB61587fE7eB23f7b0;
+    address public png_spore_png_lp = 0xad24a72ffE0466399e6F69b9332022a71408f10b;
+	  address public spore = 0x6e7f5C0b9f4432716bDd0a77a3601291b9D9e985;
+    
+    constructor(
+        address _governance,
+        address _strategist,
+        address _controller,
+        address _timelock
+    )
+        public
+        StrategyPngFarmBase(
+            spore,
+            png_spore_png_rewards,
+            png_spore_png_lp,
+            _governance,
+            _strategist,
+            _controller,
+            _timelock
+        )
+    {}
+
+    // **** State Mutations ****
+
+    function harvest() public override onlyBenevolent {
```

Comment:
Doesn't this conflict with the existing StrategyPngFarmBase? Should we just use a flattened file for this one and Avax-png-spore?